RevenueCatUI/PaywallView.swift

```diff
@@ -187,7 +187,7 @@ struct LoadedOfferingPaywallView: View {
     private let mode: PaywallViewMode
     private let fonts: PaywallFontProvider
 
-    @ObservedObject
+    @StateObject
```

Comment:
This was the main issue. We can use `@ObservedObject` on `PurchasesHandler` because it's injected, but here we create a new `IntroEligibilityViewModel`.